modules/cnf-node-tuning-operator-disabling-cpu-load-balancing-for-dpdk.adoc

```diff
@@ -0,0 +1,47 @@
+// Module included in the following assemblies:
+//
+// * scalability_and_performance/low_latency_tuning/cnf-provisioning-low-latency-workloads.adoc
+
+:_mod-docs-content-type: PROCEDURE
+[id="cnf-node-tuning-operator-disabling-cpu-load-balancing-for-dpdk_{context}"]
+= Disabling CPU load balancing in a Pod
+
+Functionality to disable or enable CPU load balancing is implemented on the CRI-O level. The code under the CRI-O disables or enables CPU load balancing only when the following requirements are met.
+
+* The pod must use the `performance-<profile-name>` runtime class. You can get the proper name by looking at the status of the performance profile, as shown here:
++
+[source,yaml]
+----
+apiVersion: performance.openshift.io/v2
+kind: PerformanceProfile
+...
+status:
+  ...
+  runtimeClass: performance-manual
+----
+
+The Node Tuning Operator is responsible for the creation of the high-performance runtime handler config snippet under relevant nodes and for creation of the high-performance runtime class under the cluster. It will have the same content as default runtime handler except it enables the CPU load balancing configuration functionality.
```

Comment:
```suggestion
The Node Tuning Operator is responsible for the creation of the high-performance runtime handler config snippet under relevant nodes and for creation of the high-performance runtime class under the cluster. It will have the same content as default runtime handler except that it enables the CPU load balancing configuration functionality.
```